### PR TITLE
guides: remove sample hsd.conf from configuration guide

### PR DESCRIPTION
**Pull request by pinheadmz**
Closes https://github.com/handshake-org/handshake-org.github.io/issues/72

the sample hsd.conf was removed in https://github.com/handshake-org/hsd/commit/299aa7b3e6189a006a03dfe1f229d171195c9028

### CODE REVIEW

**Comment by pinheadmz** [comment]
@kilpatty it was removed from hsd because the sample was so outdated and unmaintained, also had mostly default values and users were just copy+pasting it and not understanding what they were doing.

Maybe at the bottom of this guide we can add a short sample with a few default settings, so if users actually paste it, they won't affect anything, but they'll understand the syntax?

**Comment by tynes** [comment]
> Thoughts on replacing this link w/ an example to an actual sample hsd.conf? I'm happy to write one up as I actually use it pretty extensively.

Is this something you still plan on doing @kilpatty?

**Comment by pinheadmz** [comment]
@tynes I added a sample to this PR in the last commit: https://github.com/handshake-org/handshake-org.github.io/pull/78/commits/9c2cd703351300978557731d946dca092c388070

sorry i had the wrong link in my last comment

File: src/guides/config.md [hunk 1]
```diff
@@ -174,7 +174,20 @@ $ HSD_NETWORK=testnet HSD_HTTP_HOST=0.0.0.0 HSD_WALLET_HTTP_HOST=0.0.0.0 HSD_WAL
 - `wallet-auth`: Enable token auth for wallets (default: false).
 - `admin-token`: Token required if `wallet-auth` is enabled: restricts access to [all wallet admin routes.](https://handshake-org.github.io/api-docs/#wallet-admin-commands)
 
+### Example configuration file
+
+The following file could be placed at `~/.hsd/hsd.conf` for use with a mainnet hsd node.
+All the settings in the example below are defaults, meaning you do NOT need to use
+this `hsd.conf` if you do not need to change any of the values. It is
+provided only as an example of the configuration file syntax.
```
Comment (tynes) [inline src/guides/config.md:182]: Configuration options are newline delimited. The key value pairs are delimited by a colon. All configuration can be set via command line, environment variable or the config file.

**Comment by pinheadmz** [comment]
@tynes thanks dude, integrated your feedback at 000ef20c8880aa94ac946f4f9b9eab34cb565e4b

**Comment by tynes** [comment]
Looks good to me, ready to merge.

ACK 000ef20c8880aa94ac946f4f9b9eab34cb565e4b

**Pull request merged**